macos/FluentUITestViewControllers/TestButtonViewController.swift

```diff
@@ -52,6 +69,8 @@ class TestButtonViewController: NSViewController {
 		gridView.addColumn(with: disabledButtonsWithTitle)
 		gridView.addColumn(with: buttonsWithImage())
 		gridView.addColumn(with: disabledButtonsWithImage)
+		gridView.addColumn(with: buttonsWithTitleAndImage())
+		gridView.addColumn(with: disabledButtonsWithTitleAndImage)
```

Comment:
This is starting to get crowded. I could rewrite this so there's only 1 column of disabled buttons, instead of 3.